src/data/reference/hi.json

```diff
@@ -112,7 +112,11 @@
 			"returns":"p5 ऑब्जेक्ट"
 		},
 		"color":{
-			"description":"प्रकार के रंग के चर में संग्रहीत किए जाने वाले रंग बनाएं। मापदंडों को RGB या HSB मानों के रूप में व्याख्या किया जाता है, जो colorMode के अनुसार वर्तमान रंग मोड पर निर्भर करता है) ()। डिफ़ॉल्ट मोड 0 और 255 के बीच मानों के साथ RGB है, इसलिए, रंग फ़ंक्शन (255, 204, 0) एक चमकीले पीले रंग देता है। ध्यान दें कि यदि रंग () फ़ंक्शन को केवल एक मूल्य प्रदान किया जाता है, तो इसे ग्रेस्केल मूल्य के रूप में व्याख्या किया जाएगा। एक दूसरा मान जोड़ें, और इसका उपयोग अल्फा पारदर्शिता के रूप में किया जाएगा। जब तीन मान निर्दिष्ट किए जाते हैं, तो उन्हें RGB या HSB मान के रूप में व्याख्या की जाती है। चौथा मान जोड़ते समय, अल्फा पारदर्शिता लागू की जाती है। यदि केवल एक स्ट्रिंग पैरामीटर प्रदान किया जाता है, तो इसे सीएसएस संगत रंग स्ट्रिंग के रूप में व्याख्या किया जाएगा। रंगों को संख्या या व्यवस्था के रूप में संग्रहीत किया जाता है।",
+			"description":[
```

Comment:
Don't translate function names into hindi !
Revert the changes needed !

Reply:
ok